scripts/ci/build.sh

```diff
@@ -141,6 +141,7 @@ cat >>$testsrc_dir/setup.py <<EOL
                        '*.zip',
                        '*.pem',
                        '*.pfx',
+                       '*.der',
```

Comment:
If the issue is caused by a missing certificate file, why isn't the error triggered before?

Reply:
I think `*.der` should be added to L157 as well.

Reply:
done